tests/Serializer/GraphNavigatorTest.php

```diff
@@ -55,13 +77,16 @@ public function testNavigatorPassesInstanceOnSerialization()
         $context->getVisitor()->willReturn($visitor = $this->prophesize(VisitorInterface::class));
         $visitor->visitObject(Argument::cetera())->willReturn();
 
-        $visitor->startVisiting(Argument::cetera())->willReturn();
+        $visitor->startVisiting(Argument::cetera())->shouldBeCalled();
```

Comment:
This is not under test and is not the assertion of this test. Please revert the changes to `willReturn`

Reply:
👍